BGL/include/CGAL/boost/graph/alpha_expansion_graphcut.h

```diff
@@ -0,0 +1,717 @@
+#ifndef CGAL_BOOST_GRAPH_ALPHA_EXPANSION_GRAPHCUT_H
+// Copyright (c) 2014  GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0+
```

Comment:
This file is actually the culprit: its SPDX tag says LGPL-v3+ but the text of the license above says GPLv3+.

Reply:
Aaah, okay, I only modified the name of the license and forgot to change the text… I'll fix it, thanks.